packages/backend/src/services/SchedulerService/SchedulerService.ts

```diff
@@ -242,4 +247,36 @@ export class SchedulerService {
         const job = await this.schedulerModel.getJobStatus(jobId);
         return job.status;
     }
+
+    static async sendScheduler(
+        user: SessionUser,
+        scheduler: CreateSchedulerAndTargets,
+    ) {
+        if (!isUserWithOrg(user)) {
+            throw new ForbiddenError('User is not part of an organization');
+        }
+        if (!scheduler.name) {
+            throw new ParameterError(
+                'You must give a name to this scheduled delivery',
+            );
+        }
+        if (scheduler.targets.length === 0) {
+            throw new ParameterError(
+                'You must specify at least 1 destination before sending a scheduled delivery',
+            );
+        }
+
+        const slackChannels = scheduler.targets
+            .filter(isCreateSchedulerSlackTarget)
+            .map((target) => target.channel);
+        await slackClient.joinChannels(user.organizationUuid, slackChannels);
+
+        schedulerClient.addScheduledDeliveryJob(
```

Comment:
We need to validate if the user has access to the chart/dashboard.

Reply:
I also tested this on private spaces 

![image](https://github.com/lightdash/lightdash/assets/1983672/ad777a91-5da9-4787-93af-9bda35418189)

Reply:
However, this forbiddenError says it is not captured => causes backend to crash.

how do we capture errors on controllers ? is this happening on other places ?